packages/cloudflare-pages/src/cloudflare-pages.ts

```diff
@@ -25,14 +32,38 @@ export const defaultOptions: Required<CloudflarePagesOptions> = {
   external: [],
   minify: true,
   emptyOutDir: false,
+  routesJson: true,
 }
 
+type StaticRoutes = { version: number; include: string[]; exclude: string[] }
+
 export const cloudflarePagesPlugin = (options?: CloudflarePagesOptions): Plugin => {
   const virtualEntryId = 'virtual:cloudflare-pages-entry-module'
   const resolvedVirtualEntryId = '\0' + virtualEntryId
+  let config: ResolvedConfig
+  let staticRoutes: StaticRoutes
+  const staticPaths: string[] = []
 
   return {
     name: '@hono/vite-cloudflare-pages',
+    configResolved: async (resolvedConfig) => {
+      config = resolvedConfig
+      const paths = await readdir(config.publicDir, {
+        withFileTypes: true,
+      })
+      paths.forEach((p) => {
```

Comment:
you might wanna skip any dotfiles (`.startsWith('.')`)

Reply:
I suppose dotfiles, if any, should also be treated as static files. A notable example would be `/.well-known/security.txt` defined in RFC 9116.